### DISCUSSION TITLE
Uploading file name, URL converted to something I don't want

### DISCUSSION THREAD

**Discussion by kyblockstacking** [iterate-ch/cyberduck#15793] [Q&A] [answered]
My file name = "Example_Item_1+Example_Item_2"

When I try upload to my Cyberduck server, the naming is correct but the URL provided becomes "../Example_Item_1%2BExample_Item_2"

How can I get the URL to have "+" in it or have it exactly like my file name?

**Comment by Solkre** [comment, marked as answer]
That's just the way certain characters have to be shown in a URL.

https://www.w3schools.com/tags/ref_urlencode.ASP

**Comment by kyblockstacking** [comment]
Is there a way for Cyberduck to generate the URL with the "+" ?? I thought that certain special characters were supported in URLs.

I use the URL that corresponds with items, so having it not match with what is expected is an issue for me.
